backend/app/api/routes/projects.py

```
@@ -2,153 +2,289 @@
+            tenant_id=str(tenant_id),
+            user_id=str(current_user.id),
+        )
+        raise HTTPException(status_code=500, detail="Failed to list projects")
+
+
+@router.post("/projects", response_model=ProjectResponse, status_code=201)
 async def create_project(
+    request: Request,
     project_data: ProjectCreate,
-    db: AsyncSession = Depends(get_db)
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.13.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)